src/coop/rchain/lang/Rho.bnf

```diff
@@ -107,13 +109,13 @@ private comment ::= (BLOCK_COMMENT|BLOCK_DOC_COMMENT|LINE_COMMENT|LINE_DOC_COMME
 Proc4 ::= NIL | Value
 Proc3 ::= ChanRefSymbol Chan ["(" [Proc] ")"]
 Proc2 ::= Chan "!" "(" [Proc] ")"
-Proc1 ::= SUM "(" Bind "/:" Bind ")" "{" Proc "}"
-        | TOTAL "(" Bind ":\\" Bind ")" "{" Proc "}"
+Proc1 ::= SUM "(" Bind "/:" Bind ")" ProcBlock
+        | TOTAL "(" Bind ":\\" Bind ")" ProcBlock
         | FOR "(" [Bind] ")" "{" [Proc] "}"
```

Comment:
I think we can relax the syntax and allow optional `Proc` inside block.
`ProcBlock ::= "{" [Proc] "}"`

It will be less disturbing when having unfinished code. And also FOR can be defined with `ProcBlock`.

Reply:
Yes, you're right.